lib/internal/modules/esm/hooks.js

```diff
@@ -114,6 +114,25 @@ class Hooks {
     this.addCustomLoaders(userLoaders);
   }
 
+  /**
+   * Import and register custom/user-defined module loader hook(s).
+   * @param {string} urlOrSpecifier
+   */
+  async registerLoader(urlOrSpecifier) {
+    const { DefaultModuleLoader } = require('internal/modules/esm/loader')
+
+    const moduleLoader = new DefaultModuleLoader();
```

Comment:
I think this should share an existing DefaultModuleLoader instance—or at least its various caches/maps—as each instance currently has its own caches/maps.

Reply:
How I could get an existing `DefaultModuleLoader` instance?